hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupBasedLoadBalancer.java

```diff
@@ -374,17 +370,12 @@ private void generateGroupMaps(
       List<RegionInfo> regions = assignments.getValue();
       for (RegionInfo region : regions) {
         RSGroupInfo targetRSGInfo = null;
-        try {
-          String groupName = rsGroupInfoManager.getRSGroupOfTable(region.getTable());
-          if (groupName == null) {
-            LOG.info("Group not found for table " + region.getTable() + ", using default");
-            groupName = RSGroupInfo.DEFAULT_GROUP;
-          }
-          targetRSGInfo = rsGroupInfoManager.getRSGroup(groupName);
-        } catch (IOException exp) {
```

Comment:
Why this change?

Reply:
Seems the base branch is not right...

Reply:
oh. The reason is the method didn't throw IOException now. So remove the try...catch thing.